### DISCUSSION TITLE
Support for pnpm

### DISCUSSION THREAD

**Discussion by rockchalkwushock** [vercel/vercel#6342] [Ideas] [answered]
Hello I was updating this [repository](https://github.com/rockchalkwushock/codybrunner.dev/pull/40) to use [pnpm](https://pnpm.io) and when the Vercel action would run in my CI pipeline I noticed it would fail because the action logs were outputting `yarn install`. I removed the `preinstall` check that was forcing only `pnpm` to be used and this solved the failing builds.

However I wanted to know if including a check for `pnpm` and supporting it as a package manager is on the Vercel teams road map? Not sure what it would take, did a quick perusing of the source code to see where `yarn` was being used to get an idea of what was going on under the hood. I'd be happy to help with this if it's something the team wants to support.

Thanks for the great product you guys and gals provide!

Cheers 🍻

**Comment by styfle** [comment, marked as answer]
Yes, pnpm is supported now!

https://vercel.com/changelog/projects-using-pnpm-can-now-be-deployed-with-zero-configuration
